### PR TITLE
elbv2 module_util - support alb attributes and add compare method for check_mode

### PR DESCRIPTION
**Pull request by jatorcasso**
##### SUMMARY
- Add support for modifying alb specific attributes for https://github.com/ansible-collections/community.aws/pull/963
- Add compare_elb_attributes method to support check_mode

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
- community.aws.elb_application_lb
- amazon/aws/plugins/module_utils/elbv2.py

### CODE REVIEW

**Comment by jatorcasso** [comment]
recheck

File: changelogs/fragments/696-elbv2-support-alb-attributes.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - elbv2 - Add support for alb specific attributes (https://github.com/ansible-collections/amazon.aws/pull/696).
```
Comment (mandar242) [inline changelogs/fragments/696-elbv2-support-alb-attributes.yml:2]: LGTM! one small thing, if possible can you add newline at EOF here?

**Review by mandar242 — APPROVED** [review]

File: changelogs/fragments/696-elbv2-support-alb-attributes.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jatorcasso) [reply changelogs/fragments/696-elbv2-support-alb-attributes.yml]: done!

**Comment by jatorcasso** [comment]
updated PR to include adding a compare_elb_attributes() to support check_mode when modifying ALB attributes

**Review by mandar242 — APPROVED** [review]
LGTM!

**Review by jillr — APPROVED** [review]

**Comment by jillr** [comment]
regate

**Comment by jatorcasso** [comment]
regate

**Comment by jillr** [comment]
regate

**Comment by alinabuzachis** [comment]
regate

**Pull request merged**